src/modules/users/New/index.js

```diff
@@ -4,20 +4,40 @@ import PropTypes from 'prop-types'
 import Configure from './Configure'
 import Module from '../../app/Module'
 
+const admin = /admin/.test(self.location.href)
+
 class New extends React.Component {
-  title = 'Invite to Workspace'
-  menu = {
-    home: '/users',
+  // state for entire module
+  state = {
+    title: 'Invite to Workspace',
+    menu: {
+      home: '/users',
+    },
+    superuser: false,
+  }
+
+  componentWillMount() {
+    if (admin) {
+      this.setState({
+        title: 'Invite to Platform',
+        menu: {
+          nav: 'admin',
+        },
+      })
+    }
   }
 
   render() {
-    this.menu.level2 = {
+    const { menu, title } = this.state
+
+    menu.level2 = {
       text: '',
       to: this.props.location.pathname,
     }
+
     return (
-      <Module metaTitle={this.title} menu={this.menu}>
-        <Configure title={this.title} />
+      <Module metaTitle={title} menu={menu}>
+        <Configure title={title} />
```

Comment:
```suggestion
        <Configure title={title} superuser={superuser}/>
```

and pulling it out of the state too.